projects/or_scs/src/Mirza/OrgRegistry/Types.hs

```diff
@@ -171,8 +171,8 @@ instance FromJSON PartialNewOrg where
     <*> o .: "url"
 
 instance ToJSON PartialNewOrg where
-  toJSON (PartialNewOrg orgName url) = object
-    [ "name" .= orgName
+  toJSON (PartialNewOrg orgname url) = object
```

Comment:
Why the change of case here? Feels more correct before?

Reply:
Because `orgName` is an accessor function in `OrgResponse`.
```haskell
data OrgResponse = OrgResponse
  { orgGS1CompanyPrefix :: EPC.GS1CompanyPrefix
  , orgName             :: Text
  , orgUrl              :: Network.URI.URI
  }
```